src/app/courses/course-item/course-item.component.html

```diff
@@ -16,7 +16,7 @@ <h2 class="course-item__title">{{ course.title }}</h2>
         <div class="course-item__description">{{ course.description }}</div>
         <div class="course-item__actions">
             <button class="btn course-item__actions--edit icon-item"><span class="material-icons">edit</span>Edit</button>
-            <button class="btn course-item__actions--delete icon-item"><span class="material-icons">delete</span>Delete</button>
+            <button (click)="delete()" class="btn course-item__actions--delete icon-item"><span class="material-icons">delete</span>Delete</button>
```

Comment:
`onCourseDelete()` 
https://angular.io/guide/styleguide#dont-prefix-output-properties
just follow output naming conventions:
`courseDelete` - Event (EventEmitter)
`onCourseDelete` - outer event handler.

in this particular case it might be `onDeleteClick` or similar